tests/components/CardVerificationCodeElement.test.tsx

```diff
@@ -23,7 +23,7 @@ describe('CardVerificationCodeElement', () => {
   describe('mask', () => {
     test.each([3, 4])(
       'applies mask correctly with cvcLength = %d',
-      (cvcLength) => {
+      (cvcLength: any) => {
```

Comment:
Nit: using `any` can undermine the benefits of using TS, even in tests. However, it's not a blocker.

Reply:
tbh idk how this even happened lol, fixed